MainWindow.ui

```diff
@@ -823,7 +844,7 @@
      <normaloff>:/resources/timeline.png</normaloff>:/resources/timeline.png</iconset>
    </property>
    <property name="text">
-    <string>Create &amp;Timeline</string>
+    <string>Create Time Line</string>

```

Comment:
Undo this one, it's Timeline in ENIGMA. This is an ENIGMA IDE, not a GameMaker one. Please learn to spell properly.